enhancements/sig-architecture/76-addon-install-strategy/README.md

```diff
@@ -0,0 +1,189 @@
+	Placements []PlacementRef `json:"placements,omitempty"`
+}
+
+type PlacementRef struct {
+	// Name of the placement
+	// +required
+	Name string `json:"name"`
+
+	// Namespace of the placement
+	// +required
+	Namespace string `json:"space"`
+
+	// AddonTemplate is the template to generate ManagedClusterAddon
+	AddonTemplate ManagedClusterAddOnSpec `json:"addonTemplate,omitempty"`
+}
+```
+
+### Install addon according to a placement
+
+```yaml
+apiVersion: addon.open-cluster-management.io/v1alpha1
+kind: ClusterManagementAddOn
+metadata:
+  name: helloworld
+spec:
+  installStrategy:
+    type: Placements
+    placements:
+    - name: aws-placement
+      namespace: default
+```
+
+- when a cluster is added into the related `PlacementDecision`, the `ManagedClusterAddon` will be created on the cluster namespace.
+- when a cluster is removed from the related `PlacementDecision`, the `ManagedClusterAddon` will be deleted on the cluster namespace.
+
+### Install addons based on multiple placement with different configuration
+
+```yaml
+apiVersion: addon.open-cluster-management.io/v1alpha1
+kind: ClusterManagementAddOn
+metadata:
+  name: helloworld
+spec:
+  installStrategy:
+    type: Placements
+    placements:
+    - name: aws-placement
+      namespace: default
+      addonTemplate:
+        configs:
+        - group: addon.open-cluster-management.io
+          resource: addondeploymentconfigs
+          name: aws-config
+    - name: gcp-placement
+	  namespace: default
+	  addonTemplate:
+	    configs:
+		- group: addon.open-cluster-management.io
+		  resource: addondeploymentconfigs
+		  name: gcp-config
+```
+
+This will ensure that addon installed in aws uses aws-config will addon installed in gcp uses gcp config.
+
+
+### Disable addon in some clusters
+
+At first change the install strategy to `Manual`
+
+```yaml
+apiVersion: addon.open-cluster-management.io/v1alpha1
+kind: ClusterManagementAddOn
+metadata:
+  name: helloworld
+spec:
+  installStrategy:
+    type: Manual
+```
+
+Then admin can delete `ManagedClusterAddon` in certain cluster namespaces.
+
+When the user changes the strategy type from `Placement` to `Manual`, all the created are `ManagedClusterAddon` is kept. `ManagedClusterAddon`
+will not be auto-created or deleted anymore.
+
+When the user changes the strategy type from `Manual` to `Placement`, `ManagedClusterAddon` created manually by the the user will be
+re-evaluated. The spec of the `ManagedClusterAddon` will be updated based the template and the `ManagedClusterAddon` will be deleted
+if the cluster is in the related placements. 
+
+### Addon Install Controller
+
+We need to build and run a centralized addon install controller on the hub cluster. The addon
+install controller will create/delete `ManagedClusterAddon` according to the install strategy
+in related `ClusterManagementAddon`. The addon install controller will:
+- create or delete but not modify the `ManagedClusterAddon` if it exists already.
```

Comment:
the controller will not modify the `ManagedClusterAddon` if it exists even if the `ClusterManagementAddOn` type is not `Manual`? 

How to ensure 
> When the user changes the strategy type from `Manual` to `Placement`, `ManagedClusterAddon` created manually by the the user will be re-evaluated.

?